Packs/ThreatGrid/Integrations/FeedCiscoSecureMalwareAnalytics/FeedCiscoSecureMalwareAnalytics.py

```diff
@@ -0,0 +1,228 @@
+                    indicators_map[current_domain]['fields']['Tags'].append(name)
+
+                indicators_map[current_domain]['fields']['FeedRelatedIndicators'].extend(
+                    related for related in feed_related_indicators
+                    if related not in indicators_map[current_domain]['fields']['FeedRelatedIndicators'])
+
+            else:
+                indicators_map[current_domain] = {
+                    "value": current_domain,
+                    "type": 'Domain',
+                    "rawJSON": obj,
+                    "fields": {
+                        "Tags": [name],
+                        "reportedby": 'CiscoSMA',
+                        "FeedRelatedIndicators": feed_related_indicators,
+                        "Description": obj.get('description')
+                    }
+                }
+            if tlp_color:
+                indicators_map[current_domain]['fields']['trafficlightprotocol'] = tlp_color
+            if feed_tags:
+                indicators_map[current_domain]['fields']['trafficlightprotocol'] += feed_tags
+    return indicators_map
+
+
+def fetch_indicators_command(client: Client, feed_tags: list = [], tlp_color: Optional[str] = None) -> List[Dict]:
+    """Retrieves indicators from the feed
+
+    Args:
+        client: Client object with request
+        feed_tags: feed tags.
+        tlp_color: Traffic Light Protocol color.
+
+    Returns:
+        List. Processed indicator from feed.
+    """
+    indicators_map = fetch_indicators(client, feed_tags, tlp_color)
+    indicators = []
+    for key in indicators_map:
+        indicators.append(indicators_map[key])
+
+    # Each request returns one day's indicators, so we'll keep the date of the next day in IntegrationContext.
+    next_fetch = (dateparser.parse('tomorrow', settings={'RELATIVE_BASE': client.first_fetch})).isoformat()
+    demisto.setIntegrationContext({'last_fetch_date': next_fetch})
+    demisto.debug(f'{len(indicators)} XSOAR Indicators were created.')
+    return indicators
+
+
+def get_indicators_command(client: Client, args: Dict[str, str], feed_tags: list = [], tlp_color: Optional[str] = None)\
+        -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+
+    Args:
+        client: Client object with request
+        args: demisto.args()
+        feed_tags: feed tags.
+        tlp_color: Traffic Light Protocol color.
+
+    Returns:
+        Demisto Outputs.
+    """
+    limit = int(args.get('limit', 50))
+    indicators_map = fetch_indicators(client, feed_tags, tlp_color)
+
+    indicators = []
+    for key in indicators_map:
+        indicators.append({
+            "value": indicators_map[key]['value'],
+            "type": 'Domain',
+            "Tags": indicators_map[key]['fields']['Tags'],
+            "FeedRelatedIndicators": indicators_map[key]['fields']['FeedRelatedIndicators'],
+            "Description": indicators_map[key]['fields']['Description'],
```

Comment:
Shouldn't this also be in the `fetch_indicators`? as it also needed for the fetched indicators
In general, the only difference between `get_indicators_command` and `fetch_indicators_command` is that in the latest we also update the integration context and save the indicators in XSOAR (whereas in the first we don't), need to make sure this applies.